src/pages/signin/LoginForm/BaseLoginForm.js

```diff
@@ -213,6 +213,7 @@ function LoginForm(props) {
                     accessibilityLabel={translate('loginForm.phoneOrEmail')}
                     accessibilityRole={CONST.ACCESSIBILITY_ROLE.TEXT}
                     value={login}
+                    returnKeyType="go"
```

Comment:
Looks like we're using "done" in PDFPasswordForm and NewContactMethod page. So may be we use "done" for consistency?

Reply:
Is platform specific key required. I see for other pages we have simple static value as 'done'

Reply:
So android does not support 'done'?
And can we please fix these places too as it does seem like an exact issue

Reply:
And how about if we place this logic in TextInput component, instead of replicating in various components

Reply:
Okay, I am with that that as far it is consistent on all platforms. @Expensify/design could you please share your thought, if we're okay using `go` as return key on keyboard

For example: 

![275814972-c2147ee7-0b16-40d7-9d1a-0512646f4cb0](https://github.com/Expensify/App/assets/32012005/8c1c80e8-9116-445f-b843-ceb1d118cbdf)

Reply:
also cc @dubielzyk-expensify 

But yeah, having "go" here makes sense to me.

Reply:
Thank you! Let's use go 👍